src/routes/postRouter.js

```diff
@@ -0,0 +1,10 @@
+import { Router } from "express";
+import { newPost } from "../controllers/postController.js";
+import { validateSchema } from "../middleware/schemaValidator.js";
+import postSchema from "../schemas/postSchema.js";
+
+const postRouter = Router();
+
+postRouter.post("/posts", validateSchema(postSchema), newPost);
```

Comment:
vamos ter que fazer um middleware pra salvar as hashtags no corpo  de texto. acho que eu devo fazer pq to "cuidando" de hashtags.